refTemplates/05_Safety_Observability/destructive_command_guard/README.md

```
@@ -0,0 +1,6 @@
+Skeleton for destructive_command_guard
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)